### ISSUE TITLE
Not sure but I think this may have caused my apt upgrade to fail

### ISSUE THREAD

**Issue by mnorrsken** [foundObjects/pve-nag-buster#18]
After installing this, I started upgrade from web ui, the apt hook seems to execute and causes the web console to crash. I think this caused my boot/kernel/grub upgrade to crash midway because when I restarted the machine, grub could not find a kernel to boot from.

On the other node where this wasnt installed the upgrade and reboot worked.

**Comment by smiba** [comment]
Yeah, I think this did the same for me. I haven't rebooted yet but I'm honestly quite worried to...

**Comment by mnorrsken** [comment]
I think you can continue the apt update process if you run it from ssh:
`dpkg --configure --pending`
and maybe
`apt-get --fix-broken install`

**Comment by smiba** [comment]
It says there are no packages left, so hope everything ended up being installed just fine. My apt log looks worrying though, with it showing it's running the pve-nag-buster followed by "log ended"... Maybe it was the last thing left to-do and I was spared.

For now I've uncommented the pveproxy restart from the nag-buster, I'll just do this myself
